src/components/Modal/Modal.tsx

```diff
@@ -17,15 +16,13 @@ const StyledDialog = styled(Dialog)`
     }
 `;
 
-const Header = styled.div`
```

Comment:
Yep - I am researching online about whether the title of a modal should be an h1 or not. Google gives me conflicting answers - some said it is good and some don't prefer (both for accessibility sake)
What's your thought on it?

Reply:
I've done a bit of research and a modal title should be consistent with the rest of the document's headings hierarchy. In most cases this would be a h2 and it would be a subsection of what the h1 corresponds to.

Reply:
![image](https://github.com/VauxhallFoodbank/vauxhall-foodbank/assets/104017547/076c0a7f-7358-4f25-ae67-5b8fecd60096)

`Header` is used as the 'top bar' and should be a div.
The text inside should be a heading element (h1/h2/whatever makes sense) but this is passed in via `props.header`.

Reply:
I see make sense - then I should just leave it as a div then